mlflow/java/spark/pom.xml

```diff
@@ -1,13 +1,17 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
-  <artifactId>mlflow-spark</artifactId>
+  <artifactId>mlflow-spark-${scala.compat.version}</artifactId>
```

Comment:
According to convention,
we should use format like `mlflow-spark_{scala.compat.version}`